standard_template/standard/clause_1_scope.adoc

```diff
@@ -7,3 +7,5 @@ A formal JSON schema has been added for all the JSON objects identified, so that
 Unused or imprecisely specified components have been removed.
 
 Future improvements have been identified, such as the use of multiple time dimensions, or to use JSON-LD V1.1 which may give better compatibility with the OGC Coverage Implementation Schema (CIS) Version 1.1 (OGC09-146r8). These improvements are out of scope of this document.
+
+This specification does not specify any other formats othen than JSON.
```

Comment:
Maybe "...does not specify encoding of information in any format other than JSON"?